### PR TITLE
Add support for sending messages.

### PR DESCRIPTION
**Pull request by hodgestar**

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

File: go_cli/send.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+def messages_from_json(json_file):
+    for line in json_file:
+        data = json.loads(line.rstrip("\n"))
+        yield {
+            "to_addr": data["to_addr"],
+            "content": data["content"],
+            "session_event": data.get("session_event")
+        }
```
Comment (rudigiesler) [inline go_cli/send.py:71]: Do we maybe want some custom error messages for missing fields and invalid csv/json? What do these error messages look like when left like this?

**Comment by rudigiesler** [comment]
Just the one comment, looks good

**Comment by hodgestar** [comment]
Tx for asking about error cases. Ready for re-review.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**